polaris-react/loom.config.ts

```diff
@@ -74,6 +77,20 @@ function jestAdjustmentsPlugin() {
   });
 }
 
+function stylelintAdjustmentsPlugin() {
+  return createWorkspacePlugin('Polaris.Stylelint', ({tasks: {lint}}) => {
+    lint.hook(({hooks}) => {
+      hooks.configure.hook((configure) => {
+        // Modify the maximum number of allowed warnings from the default of 0
+        configure.stylelintFlags?.hook((flags) => ({
+          ...flags,
+          maxWarnings: 1000,
```

Comment:
May need to bump this number as there is already 859 warnings and this is just the start of the coverage rules we are tracking:
![image](https://user-images.githubusercontent.com/32409546/161604988-52133965-5618-46af-94d5-095651044a5d.png)

Reply:
Can always do that later when it becomes an issue (:

Reply:
Good idea, agreed 👍 Nice `⚠️` grep!